CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## CHANGELOG
 
-### 3.X.X (In progress)
+### 3.2.0 (2021-04-23)
 **SDK**
 - **Feature:** Added several chatting interfaces e.g. `sendMessageToContact`, `sendMessageToContactId` and `sendMessageToMultipleContacts` in `ChatBridgeDispatcher` for sending message to single or multiple contacts, even to a specific contact id. HostApp can get the message using `MessageToContact` object.
 - **Deprecated:** Old `getUniqueId`interface.
```

Comment:
I think the changes for `getUniqueId` need to be moved up to a new 3.X.X, right?